pkg/template/template.gen.go

```diff
@@ -23,15 +23,22 @@ import (
 )
 
 var (
-	templateInfos = map[string]Info{
+	TemplateInfos = map[string]Info{
```

Comment:
Missed this on the first pass, we're in `package template` already, so just call this `Info`. "information" is already plural, no need for the "s" either.